src/App.css

```diff
@@ -24,6 +24,24 @@
     "sidebar note-card"
     "sidebar notes-list";
 }
+.trash-container {
+  display: grid;
+  margin: 3rem 3rem 0;
+  grid-template-columns: 0.6fr 1.4fr;
+  grid-template-rows: 0.3fr 1.7fr;
+  gap: 0px 0px;
+  grid-auto-flow: row;
+  grid-template-areas: "sidebar notes-list";
+}
+.archive-container {
+  display: grid;
+  margin: 3rem 3rem 0;
+  grid-template-columns: 0.6fr 1.4fr;
+  grid-template-rows: 0.3fr 1.7fr;
+  gap: 0px 0px;
+  grid-auto-flow: row;
+  grid-template-areas: "sidebar notes-list";
+}
```

Comment:
This two classes are almost same - Its not a good practice - Give one Common Classname.